### PR TITLE
Added the products.info end point documentation

### PR DESCRIPTION
**Pull request by EOSullivanBerlin**

### CODE REVIEW

**Comment by luisaterrabrito** [comment]
Linting problems and you need to do make build to generate the apib file. Are we not going to refer to the current approach and explain the breaking changes?

File: src/06-products/products.apib [hunk 1]
```diff
@@ -24,3 +24,22 @@ Get a list of products.
...
+
+ + Response 201 (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + data (object)
+            + id: `f8ae61ec-62f3-0538-b028-185c4a5f217f` (string)
+            + name: `cookies` (string)
+            + description: `dark chocolate` (string)
+            + code: `1523999er2323712sd1342s333dsdsd38`
+            + `purchase_price`: (Money nullable)
```
Comment (dcsg) [inline src/06-products/products.apib]:
```suggestion
            + purchase_price (Money, nullable)
```

**Review by dcsg — CHANGES_REQUESTED** [review]
B: You need to build it :)

**Comment by EOSullivanBerlin** [comment]
I will build it now.

**Comment by EOSullivanBerlin** [comment]
@luisabritotl I don't know what I need to do to explain the breaking changes. Is there an example of that somewhere?

File: apiary.apib [hunk 2]
```diff
@@ -2719,6 +2719,25 @@ Get a list of products.
...
+
+ + Response 201 (application/json;charset=utf-8)
+
+    + Attributes (object)
+        + data (object)
+            + id: `f8ae61ec-62f3-0538-b028-185c4a5f217f` (string)
+            + name: `cookies` (string)
+            + description: `dark chocolate` (string)
+            + code: `1523999er2323712sd1342s333dsdsd38`
+            + `purchase_price`: (Money nullable)
```
Comment (steftrenson) [inline apiary.apib]: Looks like this wasn't build properly, it differs from products.apib

**Review by steftrenson — COMMENTED** [review]
Can you change the base branch to the 2019-03-13 branch? And at selling_price as well please.

**Comment by EOSullivanBerlin** [comment]
This now is a pretty unreadable PR. The changing of the branch base made it pretty confusing. Should I start over?

**Comment by dcsg** [comment]
Yes, please 😅

**Pull request closed without merging**